### ISSUE TITLE
[a11y] Table name - Courses

### ISSUE THREAD

**Issue by tdy** [cs-education/classTranscribe#65]
Courses URL: https://classtranscribe.ncsa.illinois.edu/courses

A data table should have a name to identify the table’s purpose to assistive technology users (see [WCAG 2.0 SC 2.4.6 Headings and Labels][1]).

Can be implemented either using HTML (i.e., `<caption>`, `summary` attribute, or `title` attribute) or ARIA (i.e., `aria-label` attribute).

[1]: https://www.w3.org/TR/UNDERSTANDING-WCAG20/navigation-mechanisms-descriptive.html
